charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -238,7 +241,10 @@ node:
   env: []
   envFrom: []
   kubeletPath: /var/lib/kubelet
+  loggingFormat: text
   logLevel: 2
+  # Log to standard error instead of files. DEPRECATED: will be removed in a future release.
+  toStderr: false
```

Comment:
I think we can fairly reasonably drop this in the helm chart unless someone specifically asks for it. No reason to provide an incentive to use known deprecated flags.

Reply:
Done.